packages/relay-compiler/core/RelayIRTransforms.js

```diff
@@ -48,6 +48,7 @@ const relayCommonTransforms: Array<IRTransform> = [
   RelayRelayDirectiveTransform.transform,
   RelayMaskTransform.transform,
   RelayDeferrableFragmentTransform.transform,
+  RelayGenerateIDFieldTransform.transform,
```

Comment:
sorry I'm not super familiar with the compiler code. If we weren't using this transform here before, where were we using it?